components/About/styles.module.scss

```diff
@@ -15,51 +15,44 @@
     background-color: #3e3f43;
 }
 
-/*.container {
-    display: flex;
-    align-items: center;
-    color: #3e3f43;
-    @media (min-width: $md) {
-        grid-template-columns: repeat(2, 1fr);
-    }
-}
-*/
-
 .container {
     display: grid;
-    align-items: center;
-    justify-content: center;
     grid-template-columns: 1fr;
     grid-gap: 10px;
     color: #3e3f43;
+
     @media (min-width: $md) {
-        grid-template-columns: repeat(2, 1fr);
+        grid-template-columns: repeat(12, 1fr);
     }
 }
 
 .myImage {
     border: 2px solid #f7dea6;
-    border-radius: 0px 120px 0px 120px;
+    border-radius: 0px 120px;
     background-color: #f7dea6;
-    width: 344px;
     height: 428px;
     margin: 0 auto;
     display: flex;
     align-items: center;
     justify-content: center;
+    padding: 20px;
+    @media (min-width: $md) {
+        grid-column: 1 / 6;
+    }
 }
 
-.caption {
-    width: 436px;
-    margin-bottom: 20px;
-}
-
-.subtitle {
-    width: 436px;
-    margin-top: 20px;
+.imageSize {
+    height: 250px;
+    width: 250px;
+    object-fit: contain;
```

Comment:
So as you can see working with images can be annoying. In this case it might be best to refactor and clean things up:

1.  Renames classes for clarity (`.myImage` -> `.imageWrap`, `.imageSize` -> `.image`)
2. Remove flex properties from `.myImage`/`.imageWrap` (i.e. `display: flex`, `align-items`, `justify-content`), update `padding` to 33px to match designs (measure from image to edge of container)
3. Update both `height` and `width` of `.imageSize`/`.image` to 100% and keep the `object-fit`. This gets the image to fill up the container/`.imageWrap` (minus the `padding`) while `object-fit` keeps it from getting warped.